### DISCUSSION TITLE
VITS fine tune loss question

### DISCUSSION THREAD

**Discussion by weidezhang** [coqui-ai/TTS#2453] [General Q&A] [answered]
Hi , 

I have basic question regarding the VITS loss. What's the diff between avg loss 1 and avg loss 0. I don't see in the code where they are defined. it seems 0 is trending down as expected but loss 1 is not. 

![Screenshot from 2023-03-23 12-58-52](https://user-images.githubusercontent.com/455617/227304101-a2b8e213-bd36-4921-b8c3-9d95e09be5e9.png)

**Comment by p0p4k** [comment, marked as answer]
So the code is in get_criterion() function in vits.py model file.
That points to the losses.py file where the gen and disc create their losses as a dict file. Seeing the order of it passed as a list, loss_0 for disc and loss_1 is for gen (which is the main loss to target performance on). This get_criterion() function is used in the trainer.py (coqui Trainer) to optimize and plot the graphs in the dashboard.
